model.py

```diff
@@ -14,9 +17,9 @@ class User(db.Model):
 
     __tablename__ = "users"
 
-    user_id = db.Column(db.Integer, autoincrement=True, primary_key=True)
-    username = db.Column(db.String(50), nullable=False, unique=True)
-    password = db.Column(db.String(50), nullable=False)
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
Each model has an `id`. No need to name it `model_id`. It was useful for learning, but now we can see it's a little redundant.

Also note that when you assign a *new* `primary_key` you don't need to set `autoincrement`. It is `True` by default.